nservicebus/upgrades/supported-versions.md

```diff
@@ -11,6 +11,8 @@ related:
 
 This page lists both currently and recently supported versions of all NServiceBus packages according to the [support policy](support-policy.md). See the [complete history](all-versions.md) for older versions.
 
+NOTE: There are [extended support plans](https://particular.net/support) available.
```

Comment:
I'm pretty sure DocsEngine won't like the link as is 😆

```suggestion
NOTE: [Extended support plans](/nservicebus/upgrades/support-policy#extended-support) are available.
```

Reply:
I just felt it would be most actionable to link to the website where they can immediately contact us, as opposed to the extended support page with more information which then again takes you to the public website to contact us. But it's debatable 🤷🏼‍♀️